ocaml/xapi/xapi_bond.ml

```diff
@@ -512,7 +514,15 @@ let destroy ~__context ~self =
       Db.PIF.destroy ~__context ~self:master;
 
       (* Clear the PIF.bond_slave_of fields of the members. *)
-      List.iter (fun slave -> Db.PIF.set_bond_slave_of ~__context ~self:slave ~value:(Ref.null)) members;
+      (* Also set the per bridge IGMP Snooping status *)
+      let igmp_snooping = Db.Pool.get_igmp_snooping_enabled ~__context ~self:(Helpers.get_pool ~__context) in
```

Comment:
We should not duplicate this code here. Once the former bond slaves get plugged, this field will be updated in the usual way.